Sources/AblyChat/RoomLifecycleManager.swift

```diff
@@ -318,12 +319,36 @@ internal actor DefaultRoomLifecycleManager<Contributor: RoomLifecycleContributor
         status.toRoomStatus
     }
 
-    internal func onChange(bufferingPolicy: BufferingPolicy) -> Subscription<RoomStatusChange> {
+    internal func onRoomStatusChange(bufferingPolicy: BufferingPolicy) -> Subscription<RoomStatusChange> {
         let subscription: Subscription<RoomStatusChange> = .init(bufferingPolicy: bufferingPolicy)
-        subscriptions.append(subscription)
+        roomStatusChangeSubscriptions.append(subscription)
         return subscription
     }
 
+    #if DEBUG
+        // TODO: clean up old subscriptions (https://github.com/ably-labs/ably-chat-swift/issues/36)
+        /// Supports the ``testsOnly_onRoomStatusChange()`` method.
+        private var statusChangeSubscriptions: [Subscription<StatusChange>] = []
```

Comment:
Looks like extra indentation bc of DEBUG

Reply:
Did I really, probably getting notifs on email so opened it on smartphone .. not sure

Reply:
I will unwatch this repo

Reply:
I mean that because of `#if DEBUG` declaration indentation is more than without it, probably you should place `#if DEBUG` without indentation to adjust this. Not sure how linting tool is going to digest it. You can also ignore this bc it's a minor suggestion.